### DISCUSSION TITLE
Change the font of header text "The Forum Helpers" (just right of the icon)?

### DISCUSSION THREAD

**Discussion by penguinmoose** [theforumhelpers/theforumhelpers.github.io#321] [Ideas]
I feel like the title ("The Forum Helpers") in the header's font is a bit odd. It looks different than the normal body text. It seems a bit too thin and I think it would look better if we change the font to something else like [Comfortaa](https://fonts.google.com/specimen/Comfortaa) bold (700). I'm not sure why the text looks different, but I just want to say that I feel like it just doesn't look very good (sorry I can't describe it much).

**Comment by Accio1** [comment]
I've converted this one to an issue also since it is a suggestion.

Issue #322
